daemon/networkdriver/portmapper/proxy.go

```diff
@@ -20,7 +20,7 @@ import (
 const userlandProxyCommandName = "docker-proxy"
 
 func init() {
-	reexec.Register(userlandProxyCommandName, execProxy)
+	//reexec.Register(userlandProxyCommandName, execProxy)
```

Comment:
This has been moved into libnetwork now - and reexec.Register complains (cannot be done twice.)